### PR TITLE
Create new events error 878

### LINKED ISSUE #878: Create new events error
### Overview
It appears that new checkin events are not being created from recurring events and an error is being console logged every time it tries. 

### Action Items
Fix the error in create event.

<details>
<summary>Create Event Error</summary>
Running createEvent:  {"name":"Not Today Team Meeting","hacknight":"Online","eventType":"Team Meeting","project":{"recruitingCategories":[],"partners":["Seeking"],"managedByUsers":[],"_id":"5edeb5c9ce228b001778fad3","name":"Not Today","description":"Not Today is an app intended to help people wait out periods of suicidal thinking without acting on their thoughts.","githubIdentifier":"202051333","projectStatus":"Active","location":"Westside","githubUrl":"https://github.com/hackforla/not-today","slackUrl":"https://hackforla.slack.com/archives/CMDEH9478","googleDriveUrl":"https://drive.google.com/drive/folders/1dRIYWa_JiBhKOVNKLJG-t7_8s_LjtKDC","googleDriveId":"1dRIYWa_JiBhKOVNKLJG-t7_8s_LjtKDC","hflaWebsiteUrl":"https://www.hackforla.org/projects/not-today","videoConferenceLink":"","lookingDescription":"- Flutter \n - Mobile Development UI/UX \n - Empathy and sensitivity with subject material and end users","createdDate":"2020-06-08T22:03:53.063Z","__v":0},"date":"2022-01-19T03:00:00.730Z","startTime":"2022-01-19T03:00:00.730Z","endTime":"2022-01-19T04:00:00.730Z","hours":1,"location":{"city":"Los Angeles","state":"CA","country":"USA"}}

REQUEST SHOULD CONTAIN CUSTOM HEADER BUT IT ISN'T FOUND

POST /api/events/ 401 0.194 ms - 12

FetchError: invalid json response body at http://localhost:4000/api/events/ reason: Unexpected token U in JSON at position 0
    at /Users/davidrubinstein/DigitalProjects/HfLA/VRMS/backend/node_modules/node-fetch/lib/index.js:272:32
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async createEvent (/Users/davidrubinstein/DigitalProjects/HfLA/VRMS/backend/workers/createRecurringEvents.js:146:33)
    at async filterAndCreateEvents (/Users/davidrubinstein/DigitalProjects/HfLA/VRMS/backend/workers/createRecurringEvents.js:102:37)
    at async runTask (/Users/davidrubinstein/DigitalProjects/HfLA/VRMS/backend/workers/createRecurringEvents.js:159:9) {
  type: 'invalid-json'
}
undefined
Today's events are created
</details>

### Resources/Instructions
I believe that this is the relevant file: 
/backend/workers/createRecurringEvents.js

This may be related to issue #273 , so check that out.

### PR DESCRIPTION
**Pull request by WalterT-MK**
Closes: #878 
More backend files that use `REACT_APP_CUSTOM_REQUEST_HEADER` instead of `CUSTOM_REQUEST_HEADER`

### CODE REVIEW

File: backend/workers/createRecurringEvents.js [hunk 1]
```diff
@@ -10,15 +10,16 @@ module.exports = (cron, fetch) => {
     let TODAY;
     const URL = process.env.NODE_ENV === 'prod' ? 'https://www.vrms.io' : 'http://localhost:4000';
 
-    const headerToSend = process.env.REACT_APP_CUSTOM_REQUEST_HEADER;
-
+    const headerToSend = process.env.CUSTOM_REQUEST_HEADER;
+    console.log(headerToSend);
```
Comment (FoxShaunR) [inline backend/workers/createRecurringEvents.js]:
```suggestion
```

Comment (FoxShaunR) [reply backend/workers/createRecurringEvents.js]: Forgot to remove some debug code

File: backend/workers/createRecurringEvents.js [hunk 2]
```diff
@@ -139,18 +140,20 @@ module.exports = (cron, fetch) => {
                 },
                 body: jsonEvent
             }
+
             console.log('Running createEvent: ', jsonEvent);
 
             try {
-                const response = await fetch(`${URL}/api/events/`, options); 
+                // console.log(headerToSend);
```
Comment (FoxShaunR) [inline backend/workers/createRecurringEvents.js]:
```suggestion
```

File: backend/workers/createRecurringEvents.js [hunk 3]
```diff
@@ -159,12 +162,12 @@ module.exports = (cron, fetch) => {
...
         console.log("Today's events are created");
-    
+
     };
 
     const scheduledTask = cron.schedule('*/30 * * * *', () => {
         runTask();
     });
-
+     runTask();
```
Comment (FoxShaunR) [inline backend/workers/createRecurringEvents.js]:
```suggestion
```

More left over debug code

**Comment by WalterT-MK** [comment]
new pull request has been made, closing this pull request

**Review by jbubar — COMMENTED** [review]
I am ok with this from my read through, but I have not run it yet

**Review by FoxShaunR — APPROVED** [review]

**Pull request merged**